activestorage/test/models/attachments_test.rb

```diff
@@ -68,7 +68,7 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
   test "attach new blobs" do
     @user.highlights.attach(
       { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" },
-      { io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg" })
+      io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg")
```

Comment:
@rafaelfranca I think we should consider reverting 77be872819d6e9fd63f4253398dbfd6990166a7c, and fixing the other places affected. I didn't realise such an option existed (or maybe it didn't at the time?), and definitely recall that rule making some undesirable changes when it was first applied.

https://github.com/matthewd/rails/commit/962a01d1e67b9b76d23b8b61a760254d3622d80f feels more good than bad to me... there are some methods that take options after a hash parameter (e.g. `select` and friends), but most seem to be of the "two equally-important hashes" variety.